impl/client/src/main/javascript/web/pentaho/visual/models/Donut.js

```diff
@@ -17,13 +17,27 @@ define([
   "pentaho/module!_",
   "./Pie",
   "pentaho/i18n!./i18n/model"
-], function(module, BaseModel, bundle) {
+], function (module, BaseModel, bundle) {
 
   "use strict";
 
   return BaseModel.extend({
     $type: {
-      id: module.id
+      id: module.id,
+
+      props: [
+        {
+          name: "explodedSliceRadius",
+          valueType: "number",
+          defaultValue: "2"
+        },
+        {
+          name: "slice_innerRadiusEx",
+          label: "Inner Radius",
+          valueType: "number",
+          defaultValue: "50"
```

Comment:
Name does not seem appropriate. Consider `sliceInnerRadius`.
Label not localized.
It seems we're changing the defaults of these two properties. At least DET uses this. Could you clarify what's the actual impact?  Previously, it was possible to configure these via extension. If so, then, usually extensions override model options, IIRC. Example:
https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/ced4b4bb0f5a901dff0e0d7468d8993d17ee5380/impl/client/src/main/javascript/web/pentaho/ccc/visual/Abstract.js#L1181-L1183

Reply:
Only partially addressed.